src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (a) => (b) => {
+    count++;
+
+    if (count >= 3 && count % 2 === 0) {
+      return 'Bzzz... Error!';
+    }
+
+    return a + b;
+  };
```

Comment:
This syntax might be a bit confusing
```suggestion
  const getSum = (a) => (
    (b) => {
      count++;

      if (count >= 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return a + b;
    }
  );
```